docs/electron.rst

```diff
@@ -156,6 +157,19 @@ is not required.
 ``org.electronjs.Electron2.BaseApp`` since ``branch/23.08`` comes with
 ``libnotify>=0.8.0``
 
+.. _use-correct-desktop-filename:
+
+Using correct desktop file name
+-------------------------------
+
+It's important for Linux applications to set the correct desktop file name. If not, it can led to problems e.g. a missing icon under Wayland.
```

Comment:
```suggestion
It's important for Linux applications to set the correct desktop file name. If not, it can lead to problems e.g. a missing icon under Wayland.
```